ament_cpplint/ament_cpplint/main.py

```diff
@@ -77,6 +77,10 @@ def main(argv=sys.argv[1:]):
     parser.add_argument(
         '--root', type=str,
         help='The --root option for cpplint')
+    parser.add_argument(
+        '--exclude', default=[''],
```

Comment:
The default value should be an empty list and not contain an empty string. That also avoids the condition below `not excl`.